### ISSUE TITLE
errors in minifier are not propagated properly

### ISSUE THREAD

**Issue by max06** [NodeBB/NodeBB#6241]
- **NodeBB version:** 1.6.1
- **Database type:** redis
- **Database version:** 3.0.7

- **Exact steps to cause this issue:**

Since this morning, a new installation fails when I try to build the styles.
When executing `./nodebb build`:
```
15/1 09:43:27 [13551] - info: [build] Building in parallel mode
15/1 09:43:27 [13551] - info: [build]         plugin static dirs  build started
15/1 09:43:27 [13551] - info: [build]          requirejs modules  build started
15/1 09:43:27 [13551] - info: [build]           client js bundle  build started
15/1 09:43:27 [13551] - info: [build]            admin js bundle  build started
15/1 09:43:28 [13551] - info: [build]         client side styles  build started
15/1 09:43:28 [13551] - info: [build] admin control panel styles  build started
15/1 09:43:28 [13551] - info: [build]                  templates  build started
15/1 09:43:28 [13551] - info: [build]                  languages  build started
15/1 09:43:28 [13551] - info: [build]                     sounds  build started
15/1 09:43:28 [13551] - info: [build]                     sounds  build completed in 0.431sec
15/1 09:43:28 [13551] - info: [build]         plugin static dirs  build completed in 0.65sec
15/1 09:43:30 [13551] - info: [build]                  templates  build completed in 2.866sec
15/1 09:43:31 [13551] - error: [build] admin control panel styles  build failed
15/1 09:43:31 [13551] - error: [build] Encountered error during build step: undefined
15/1 09:43:31 [13551] - error:
```
The error output is not really helpful. After that there are no styes, no templates, no js-bundles.

I figured out that the dependency `browserslist` (a dep of `autoprefixer`) was updated by a patchlevel this morning. Pinning it back to version `2.11.1` in nodebb fixes that issue. 

- **What you expected:**
Compiled assets or more detailed error output
- **What happened instead:**
Guessing what happened.

**Comment by ai** [comment]
Maybe the error is inside Browserslist, but anyway we need a better error output here for the future. Without better output, I can’t fix Browserslist.

**Comment by max06** [comment]
Issue is solved in browserslist@2.11.3. Still a better error output here might be welcome.
